components/collector/src/source_collectors/visual_studio_trx/tests.py

```diff
@@ -0,0 +1,47 @@
+"""Visual Studio TRX tests collector."""
+
+import re
+from typing import cast
+from xml.etree.ElementTree import Element  # nosec # Element is not available from defusedxml, but only used as type
```

Comment:
Perhaps only import if typing.TYPE_CHECKING ?

Reply:
Alas, then the unit tests fail:

```
    def __entity(test: Element, result: str, namespaces: Namespaces) -> Entity:
                       ^^^^^^^
NameError: name 'Element' is not defined
```